Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/WriteTelemetryTaskTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿/*
+ * SonarScanner for .NET
+ * Copyright (C) 2016-2025 SonarSource SA
+ * mailto: info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SonarScanner.MSBuild.Tasks.IntegrationTest.TargetsTests;
+
+[TestClass]
+public class WriteTelemetryTaskTests
+{
+    private const string TelemetryFileName = "Telemetry.json";
+
+    public TestContext TestContext { get; set; }
+
+    [TestCategory(TestCategories.NoUnixNeedsReview)]
```

Comment:
Can be removed after https://github.com/SonarSource/sonar-scanner-msbuild/pull/2506 is merged